bogota-july-2018/Calculator-JS/DaniloPlazas/Basic Function/scripts.js

```diff
@@ -0,0 +1,30 @@
+function doArithmeticOperation(number1, number2, operation) {
+
+  var arithmeticResult = 0.0;
+
+  switch (operation) {
+    case "+":
+      arithmeticResult = number1 + number2;
+      break;
+    case "-":
+      arithmeticResult = number1 - number2;
+      break;
+    case "*":
+      arithmeticResult = number1 * number2;
+      break;
+    case "/":
+      arithmeticResult = number1 / number2;
+      break;
+    default:
+      arithmeticResult = 0;
+
+  }
+
+  return arithmeticResult || 0;
+
+}
+
+console.log(doArithmeticOperation(0,100,"+"));
```

Comment:
`console.log(doArithmeticOperation('a', '/',"+"));` this is failling, please check the data that is introduced to the function and provide accurate error messages :)

Reply:
For the operator sign as well